### DISCUSSION TITLE
Too old container downloads …

### DISCUSSION THREAD

**Discussion by FreeSynergy** [keycloak/keycloak#35582] [Feedback]
My Admin asked me, what is wrong with the Docker Container, KeyCloak offers officialy?
They are more than 3 years old …
Is this right?

![Screenshot_20241204-022215](https://github.com/user-attachments/assets/c3042cc9-ce34-4456-afa8-fc0730b31ffb)

**Comment by stianst** [comment]
That's recent builds, not published tags. Look at published tags instead and you'll see the latest is days, not years old.

**Reply by FreeSynergy** [reply]
Thank you …
